examples/web/index.html

```diff
@@ -73,16 +73,29 @@ <h3 class="typography-heading-small-text mt-1100 mb-800">Square Button</h3>
       <section>
         <div class="Container">
           <h2 class="typography-heading-medium-text mt-1100 mb-800">Tag</h2>
-          <span class="Tag Tag--default-light">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--default-dark">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--info-light">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--info-dark">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--success-light">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--success-dark">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--warning-light">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--warning-dark">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--danger-light">I'm tag. I feel like filling marketing pages :-)</span>
-          <span class="Tag Tag--danger-dark">I'm tag. I feel like filling marketing pages :-)</span>
+          <span class="Tag Tag--default Tag--light Tag--small">I'm tag. I feel like filling marketing pages :-)</span>
```

Comment:
Just for information. We are deprecating this examples in favor of new dev-stack, which is ready to merge. Maybe it will be better to do this in a new dev-stack. What do you think?

Reply:
As it is not merged, I would keep It here and who merges last will fix this.

Reply:
I copypasted the example in the component index as well.